index.js

```diff
@@ -33,10 +34,12 @@ async function postGraphilePlugin (fastify, opts) {
   const {
     database,
     schemas = 'public',
-    settings = defaultSettings
+    settings
   } = opts
 
-  const middleware = postgraphile(database, schemas, settings)
+  const deepMergedSettings = mergeWith({}, defaultSettings, settings, concatNestedArrays)
+
```

Comment:
This change will merge in settings passed in from an implementation of `fastify-postgraphile` with the default settings already defined. Boolean or string default settings will be replaced by the user settings but anything defined in the `appendPlugins` array will be concatenated with the default `appendPlugins`.

The drawback to this approach is that there is no way to remove the default implementation of the `simplifyInflector` plugin. 

Should we define any plugins at all in the default settings and require that all plugins be installed and passed in by the specific usage of `fastify-postgraphile`?

Should we continue to define the default `simplifyInflector` plugin but merge user settings by overwriting the entire default `appendPlugins` array with anything passed in? This would mean that the user would need to both add their own plugin(s) and re-add the `simplifyInflector` plugin.

What is more ergonomic for users of `fastify-postgraphile`? 

Thoughts?